src/components/PopupWithForm.js

```diff
@@ -16,6 +18,14 @@ export default class PopupWithForm extends Popup {
     return inputValues;
   }
 
+  setIsLoading(isLoading) {
+    if (isLoading) {
+      this._submitButton.textContent = "Saving...";
+    } else {
+      this._submitButton.textContent = this._submitButtonText;
+    }
+  }
+
```

Comment:
```js


  close() {
    super.close();
  }
```

Please, delete the `close` method from `PopupWithForm`

You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.